perf/src/main/java/tachyon/perf/benchmark/Operators.java

```diff
@@ -143,7 +143,7 @@ public static long readSingleFile(PerfFileSystem fs, String filePath, int buffer
     byte[] content = new byte[bufferSize];
     InputStream is = fs.open(filePath, readType);
     int onceLen = is.read(content);
-    while (onceLen != -1) {
+    while (onceLen > 0) {
```

Comment:
In tachyon.client, the read returns 0 when the byte[]'s length is 0 or nothing to read.
Here use ">0" instead of "!= -1" to avoid the infinite loop (The block is evicted but TachyonMaster still hold the file's info, then the read always returns 0).